go/internal/initutil/manager.go

```diff
@@ -66,6 +66,14 @@ type Manager struct {
 			// The netry : `:none:` will disable all rdvp servers.
 			RdvpMaddrs flagStringSlice
 
+			// Tor
+			Tor struct {
+				Enabled       bool
+				BinaryPath    string // if "" that mean embedded must be used.
+				DontSetListen bool
+			}
+			AnonymityMode bool // Desactivate other stuff to enforce anonymity.
```

Comment:
I like the idea of adding this, but I prefer to use a `Preset string` instead of `Anonymity bool`, so we can have more than two presets

Reply:
> I like the idea of adding this, but I prefer to use a `Preset string` instead of `Anonymity bool`, so we can have more than two presets

This is more than a preset, this sets `allowtcpdial` on tor, disable all other transports (except circuit) and disable single hop (not done yet).